packages/strapi-plugin-content-manager/admin/src/hooks/useFetchContentTypeLayout/utils/formatLayouts.js

```diff
@@ -66,49 +127,15 @@ const generateRelationQueryInfosForComponents = (obj, fieldName, ctUid, models)
   return queryInfos;
 };
 
-// editRelations is an array of strings...
-const formatEditRelationsLayoutWithMetas = (obj, models) => {
-  const formatted = obj.layouts.editRelations.reduce((acc, current) => {
-    const fieldSchema = get(obj, ['schema', 'attributes', current], {});
-    const metadatas = get(obj, ['metadatas', current, 'edit'], {});
-    const size = 6;
-
-    const queryInfos = generateRelationQueryInfos(obj, current, models);
-
-    acc.push({
-      name: current,
-      size,
-      fieldSchema,
-      metadatas,
-      queryInfos,
-    });
-
-    return acc;
-  }, []);
-
-  return formatted;
-};
-
-const formatLayouts = (data, models) => {
-  const formattedCTEditLayout = formatLayoutWithMetas(data.contentType, models);
-  const ctUid = data.contentType.uid;
-  const formattedEditRelationsLayout = formatEditRelationsLayoutWithMetas(data.contentType, models);
-
-  set(data, ['contentType', 'layouts', 'edit'], formattedCTEditLayout);
-  set(data, ['contentType', 'layouts', 'editRelations'], formattedEditRelationsLayout);
-
-  Object.keys(data.components).forEach(compoUID => {
-    const formattedCompoEditLayout = formatLayoutWithMetas(
-      data.components[compoUID],
-      ctUid,
-      models
-    );
-
-    set(data, ['components', compoUID, 'layouts', 'edit'], formattedCompoEditLayout);
-  });
-
-  return data;
-};
+const getDisplayedModels = models =>
+  models.filter(model => model.isDisplayed).map(({ uid }) => uid);
 
 export default formatLayouts;
-export { formatEditRelationsLayoutWithMetas, formatLayoutWithMetas };
+export {
+  formatEditRelationsLayoutWithMetas,
+  formatLayoutWithMetas,
+  formatListLayoutWithMetas,
+  generateRelationQueryInfos,
+  generateRelationQueryInfosForComponents,
+  getDisplayedModels,
+};
```

Comment:
This filename is formatLayouts.js but there is more than this function :D maybe e can rename it

Reply:
The default export is formatLayout. I have chosen to split the main one into several small functions so the tests are easier to write and to maintain so I don't think renaming this file is needed here.
What do you think?